pandas/core/internals/blocks.py

```diff
@@ -669,7 +669,7 @@ def convert(
 
         return self.copy() if copy else self
 
-    def _can_hold_element(self, element):
+    def _can_hold_element(self, element) -> bool:
```

Comment:
`element` here should be `Any` right? If so I think OK to type as an explicit indication

Reply:
Note this comment is applicable in a lot of the signatures here

Reply:
will do